src/Classes/NIPSWeb/NIPSWeb_ManagedItem.php

```diff
@@ -214,15 +214,22 @@ public static function storeItem($tmpid, $title)
         }
         $dbfile = $item->getFolder().'/'.$item->getID();
 
-        //Convert it with ffmpeg
+        //Convert it with ffmpeg/avconv
+        if (`which ffmpeg`) {
```

Comment:
Eww indeed. Feel free to look at the various ways of calling things in PHP to investigate. This is more-or-less duplication of what's already in Track.